dotnet-desktop-guide/net/wpf/whats-new/net90.md

```diff
@@ -0,0 +1,123 @@
+
+To apply a theme mode for the whole application, set the `ThemeMode` property on the `Application` type. To apply it to a single window, set `ThemeMode` on the `Window` type.
+
+For example, style the entire application based on the current light or dark theme set by Windows:
+
+:::code language="csharp" source=".\snippets\net90\csharp\App.xaml" range="1-6" highlight="6":::
+
+Here's an example of forcing the light theme, regardless of the theme set by Windows:
+
+:::code language="csharp" source=".\snippets\net90\csharp\LightWindow.xaml" range="1-6" highlight="6":::
+
+If the `ThemeMode` is set to any value other than `None` at the application-level, `None` can no longer be applied at the window-level.
+
+`ThemeMode` is designed to respect the settings set by a Fluent Dictionary, allowing you to customize the Fluent theme.
+
+### Set in code
+
+Support for changing setting the `ThemeMode` in code is currently an experimental feature. Accessing the `ThemeMode` property in code generates error **WPF0001**, preventing access to the API. Suppress the error to access to the API.
+
+> [!WARNING]
+> This API is experimental and subject to change.
+
+First, add the following `PropertyGroup` element to your project file to suppress the error:
+
+:::code language="xml" source=".\snippets\net90\csharp\MyWpfProject.csproj" id="NoWarn":::
+
+> [!TIP]
+> You can use the `#pragma warning disable WPF0001` directive to suppress the error where it occurs instead of disabling it for the entire project.
+
+Next, set either the `ThemeMode` property at the application-level or window-level:
+
+:::code language="csharp" source=".\snippets\net90\csharp\MainWindow.xaml.cs" id="ThemeMode":::
+
+## Support for Windows accent color
+
+Windows 10 introduced a user-selectable accent color that's used in providing a personal touch or calling out a specific visual element. WPF now supports the user-selected accent color.
+
+The visual color is available as a `System.Windows.Media.Color`, `System.Windows.Media.SolidColorBrush`, or `System.Windows.ResourceKey`. Along with the color itself,  light and dark shades of the accent color are available. These are accessed through `System.Windows.SystemColors`:
+
+|         | Color               | Brush                    | Resource Key           |
+|---------|---------------------|--------------------------|------------------------|
+| Accent  | `AccentColor`       | `AccentColorBrush`       | `AccentColorKey`       |
+| Light 1 | `AccentColorLight1` | `AccentColorLight1Brush` | `AccentColorLight1Key` |
+| Light 2 | `AccentColorLight2` | `AccentColorLight2Brush` | `AccentColorLight2Key` |
+| Light 3 | `AccentColorLight3` | `AccentColorLight3Brush` | `AccentColorLight3Key` |
+| Dark 1  | `AccentColorDark1`  | `AccentColorDark1Brush`  | `AccentColorDark1Key`  |
+| Dark 2  | `AccentColorDark2`  | `AccentColorDark2Brush`  | `AccentColorDark2Key`  |
+| Dark 3  | `AccentColorDark3`  | `AccentColorDark3Brush`  | `AccentColorDark3Key`  |

```

Comment:
We can add the following "Brush Resource Key" in the list : "<Brush Name>Key"

Like AccentColorBrushKey, AccentColorLight1BrushKey, and so on....

Reply:
https://github.com/dotnet/wpf/issues/9295 - here is the list for the last set of resource keys